test/cpp/src/498-cache-plain-printer.cc

```diff
@@ -8,12 +8,12 @@ TEST_CASE("An empty cache stat block prints all zeros") {
   given.name = "test_cache";
 
   std::vector<std::string> expected{
-    "test_cache TOTAL        ACCESS:          0 HIT:          0 MISS:          0",
-    "test_cache LOAD         ACCESS:          0 HIT:          0 MISS:          0",
-    "test_cache RFO          ACCESS:          0 HIT:          0 MISS:          0",
-    "test_cache PREFETCH     ACCESS:          0 HIT:          0 MISS:          0",
-    "test_cache WRITE        ACCESS:          0 HIT:          0 MISS:          0",
-    "test_cache TRANSLATION  ACCESS:          0 HIT:          0 MISS:          0",
+    "test_cache TOTAL        ACCESS:          0 HIT:          0 MISS:          0 DOWNSTREAM:          0",
+    "test_cache LOAD         ACCESS:          0 HIT:          0 MISS:          0 DOWNSTREAM:          0",
+    "test_cache RFO          ACCESS:          0 HIT:          0 MISS:          0 DOWNSTREAM:          0",
+    "test_cache PREFETCH     ACCESS:          0 HIT:          0 MISS:          0 DOWNSTREAM:          0",
+    "test_cache WRITE        ACCESS:          0 HIT:          0 MISS:          0 DOWNSTREAM:          0",
+    "test_cache TRANSLATION  ACCESS:          0 HIT:          0 MISS:          0 DOWNSTREAM:          0",
```

Comment:
I have mixed feelings about this reporting. On the one hand, I have wanted for a long time to add in some kind of "MSHR merged" indicator. On the other hand, I think that "downstream" is either not the right name or not the right statistic to be showing to the user.

This information, by your admission, is already shown in the lower-level accesses, and we shouldn't show redundant information. We get so many questions already about how to interpret these statistics that I want to make sure any changes are in the direction of increased clarity.

Reply:
I think I get more questions about why misses don't equal accesses in the downstream cache more than anything else. We could report the # of merges instead, which would be misses - downstream.

Reply:
I would rather do that, because that says positively what happened to each packet. However, if we simply call it `MERGED`, then the question will be "merged with what?", so we need a name like `MSHR_MERGE` or something.